index.html

```diff
@@ -0,0 +1,539 @@
+              </path>
+            </g>
+          </svg>
+        </div>
+        <div>
+          <p>Seriously, red?</p>
+        </div>
+      </div>
+      <div class="vitaly authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/vitaly-friedman.jpg" alt=""></div>
+          <div>
+            <p>Vitaly Friedman <em>wrote</em></p>
+          </div>
+        </div>
+        <div>
+          <p>Make honest design work for digital business, with ethical design handbook <span class="commentp">Leave a
+              comment</span></p>
+          <ul>
+            <li>#Privacy <sup>13</sup></li>
+            <li>#UX <sup>1</sup></li>
+            <li>#Smashing books <sup>37</sup></li>
+          </ul>
+          <p class="arrow">→</p>
+        </div>
+      </div>
+      <div class="leonardo authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/leonardo-losoviz-portrait.jpg" alt=""></div>
+          <div>
+            <p>Vitaly Friedman <em> wrote</em></p>
+          </div>
+        </div>
+        <div>
+          <p>Make honest design work for digital business, with ethical design handbook <span class="commentp">Leave a
+              comment</span></p>
+          <ul>
+            <li>#Privacy <sup>13</sup></li>
+            <li>#UX <sup>1</sup></li>
+            <li>#Smashing books <sup>37</sup></li>
+          </ul>
+          <p class="arrow">→</p>
+        </div>
+      </div>
+      <div class="drew authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/drew-mclellan-profile-pic.jpg" alt=""></div>
+          <div>
+            <p>Vitaly Friedman <em> wrote</em></p>
+          </div>
+        </div>
+        <div>
+          <p>Make honest design work for digital business, with ethical design handbook <span class="commentp">Leave a
+              comment</span></p>
+          <ul>
+            <li>#Privacy <sup>13</sup></li>
+            <li>#UX <sup>1</sup></li>
+            <li>#Smashing books <sup>37</sup></li>
+          </ul>
+          <p class="arrow">→</p>
+        </div>
+      </div>
+      <div class="barry authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/barry-pollard-200x200.jpg" alt=""></div>
+          <div>
+            <p>Vitaly Friedman <em> wrote</em></p>
+          </div>
+        </div>
+        <div>
+          <p>Make honest design work for digital business, with ethical design handbook <span class="commentp">Leave a
+              comment</span></p>
+          <ul>
+            <li>#Privacy <sup>13</sup></li>
+            <li>#UX <sup>1</sup></li>
+            <li>#Smashing books <sup>37</sup></li>
+          </ul>
+          <p class="arrow">→</p>
+        </div>
+      </div>
+    </div>
+  </main>
+  <article class="latest-posts">
```

Comment:
The latest post section it's different from the page. The first row should have 3 articles of the same size but the next two have one a little larger.

Reply:
Done!